### PR TITLE
Move Secret Scan from Beta

### PR DESCRIPTION
**Pull request by dev-yakuza**
> Explain a summary, purpose, or background for this change.

The `Secret Scan` is ready to launch. So, I remove the `Beta` flag to release.

> Refer related issues or pull requests (e.g. `Close #123` or `None`).

https://github.com/sider/sideci/issues/11384

> Check the following if needed.

- [x] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file.
 
 [Full diff](https://github.com/sider/runners/compare/0.51.2...HEAD)
 
+- **Secret Scan** Change Beta to Release [#11490](https://github.com/sider/sideci/pull/11490)
```
Comment (ybiquitous) [inline CHANGELOG.md:9]:
```suggestion
- **Secret Scan** Change Beta to Release [#2529](https://github.com/sider/runners/pull/2529)
```

Also, please update the PR title.

Comment (dev-yakuza) [reply CHANGELOG.md]: Thanks for your review.
Fixed it on https://github.com/sider/runners/pull/2529/commits/867c6713b91553780007049a5172bc698e48fa6a and I changed the PR title.

**Review by ybiquitous — APPROVED** [review]
👍🏼

**Comment by dev-yakuza** [comment]
This PR can be merged after https://github.com/sider/sideci/issues/11505

**Comment by sumyapp** [comment]
https://docs.google.com/document/d/1IMBsrCWCkPejOHRc25TOfgspJm7-FBNyGOxJQt4JZT4/edit#

**Comment by sumyapp** [comment]
- [ ] Publish press release(EN)
- [ ] Publish press release(JP)
- [ ] Publish blog post(JP)
- [ ] Publish blog post(EN)

File: CHANGELOG.md [hunk 2]
```diff
@@ -49,6 +49,8 @@ Misc:
 - **PHPMD** Enable new recommended configuration [#2536](https://github.com/sider/runners/pull/2536)
 - **PMD CPD** Remove codefragment from results [#2525](https://github.com/sider/runners/pull/2525)
 
+- **Secret Scan** Change Beta to Release [#2529](https://github.com/sider/runners/pull/2529)
+
```
Comment (ybiquitous) [inline CHANGELOG.md:53]: Oh, this position is incorrection. 😓

Comment (dev-yakuza) [reply CHANGELOG.md:53]: Sorry, I will make a new PR 😭

**Pull request merged**